hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/HeapDataTest.java

```diff
@@ -56,4 +58,11 @@ public void copyTo() {
 
         assertEquals(new String(inputBytes), new String(bytes));
     }
+
+    @Test
+    public void toHeapData() {
+        HeapData anotherHeapData = new HeapData();
+        Data toOnHeap = anotherHeapData.toHeap();
+        assertInstanceOf(HeapData.class, toOnHeap);
```

Comment:
I would do an assertSame.

Reply:
Since a new object is created for HeapData assertSame will not work. I have changed it to assertEquals.